jane/doc/extensions/_05-kinds/syntax.md

```diff
@@ -168,13 +168,18 @@ The abbreviations defined in the language are as follows:
     [externality][].
 
 * `immutable_data =
-     value mod many contended portable unyielding immutable stateless`
+     value mod many contended portable unyielding immutable stateless non_float`
 
     This is a suitable kind for plain old data that is immutable. By "plain
     old data", we mean that values of types of this kind contain no pointers to
     functions. The type `string` has this kind.
 
-* `mutable_data = value mod many portable unyielding stateless`
+* `sync_data = value mod many contended portable unyielding stateless non_float`
+
+   This is a suitable kind of plain old data that can only be mutated
+   synchronously, similar to the `Sync` trait in Rust.
```

Comment:
I originally interpreted this to be the opposite of what you probably mean. I think this is data such that all mutable fields are always synchronized, or something. Is that right?

Reply:
Are mutable fields in a record stored in a `Capsule.Data.t` synchronized? I suppose.

Reply:
Is it better now?

Reply:
Superseded by other comments.